data/types/food/food_test.go

```diff
@@ -22,12 +22,14 @@ func NewMeta() interface{} {
 	}
 }
 
-func NewFood() *food.Food {
+func NewFood(ingredientArrayDepth int) *food.Food {
```

Comment:
depth or length??

Reply:
Depth. This test function creates a hierarchy of ingredients. This parameter limits the total overall depth (or otherwise it recursively creates more and more and eventually blows the stack).